compiler/codegen/codegen.cpp

```diff
@@ -1226,25 +1226,6 @@ static void protectNameFromC(Symbol* sym) {
     return;
   }
 
-  //
-  // For now, we only rename our user and standard symbols.  Internal
-  // modules symbols should arguably similarly be protected, to ensure
-  // that we haven't inadvertently used a name that some user library
-  // will; most file-level symbols should be protected by 'chpl_' or
-  // somesuch, but of course local symbols may not be, and can cause
-  // conflicts (at present, a local variable named 'socket' would).
-  // The challenges to handling MOD_INTERNAL symbols in the same way
-  // today is that things like chpl_string and uint64_t should not be
-  // renamed, and should arguably have FLAG_EXTERN on them; however,
-  // putting it on them causes it to bleed over onto type aliases in a
-  // way that breaks things and wasn't easy to fix.  So this remains
-  // a TODO (currently in Brad's court).
-  //
-  ModuleSymbol* symMod = sym->getModule();
-  if (symMod->modTag == MOD_INTERNAL) {
-    return;
-  }
-
   //
   // If this symbol is exported of an extern symbol then someone
```

Comment:
Optional: while here, fix the above line, ex. `... is exported or extern, then ...`

Reply:
Thanks!